ciongke/test-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
What's important is that `apt-get update` and `apt-get install` are in a single `RUN`, which I obeyed here.